cli.js

```diff
@@ -49,6 +49,8 @@ var argv = require('optimist')
       throw 'Meta schema file "' + args.s + '" does not exist!';
     }
   })
+  .alias('h', 'header')
+  .describe('h', 'if the value is false the header will be skipped')
```

Comment:
Not sure that it works like you intended/is documented.The default of the parameter should be true, and you can disable it by calling `jsonschema2md --no-header` or `jsonschema2md -h false`.

Here is how to declare a boolean flag: https://yargs.js.org/docs/#api-booleankey